lynx/src/test/java/com/github/pedrovgs/lynx/LynxConfigTest.java

```diff
@@ -16,30 +16,50 @@
 
 package com.github.pedrovgs.lynx;
 
+import com.github.pedrovgs.lynx.model.TraceLevel;
+import org.junit.Rule;
 import org.junit.Test;
+import org.junit.rules.ExpectedException;
 
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertNotNull;
 import static org.junit.Assert.assertTrue;
 
 /**
  * @author Pedro Vicente Gómez Sánchez.
  */
 public class LynxConfigTest {
+  @Rule
+  public ExpectedException exception = ExpectedException.none();
 
   private static final String ANY_FILTER = "AnyFilter";
 
+  @Test public void setFilterNullThrowsNPE() {
+    exception.expect(NullPointerException.class);
+    exception.expectMessage("filter can't be null");
+    LynxConfig lynxConfig = new LynxConfig();
+    lynxConfig.setFilter(null);
+  }
+
+  @Test public void setFilterTraceLevelNullThrowsNPE() {
+    exception.expect(NullPointerException.class);
+    exception.expectMessage("filterTraceLevel can't be null");
+    LynxConfig lynxConfig = new LynxConfig();
+    lynxConfig.setFilterTraceLevel(null);
+  }
```

Comment:
You can use just a test like this instead of use ExpectedExceptionN:

```java

@Test(expected = IllegalArgumentException.class) public void setFilterNullThrowsIllegalArgumentException(){
    lynxConfig.setFilter(null);
}

```